### PR TITLE
adding markdown files to git

### PR DESCRIPTION
**Pull request by tmccourt**
Added the actual docs

### CODE REVIEW

File: manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+- Openssl version 0.9.8 or higher and libssl-devel packages
+- GCC compiler suite. (tested with 2.9x, 3.x and 4.x series)
+
+If you are compiling with higher optimization levels than the default one (-O
+for gcc), be aware that there have been reports of misoptimizations. The
+build system of ClamAV only checks for bugs affecting the default settings,
+it is your responsibility to check that your compiler version doesn’t have any
+bugs.
+
+- GNU make (gmake)
```
Comment (micah-at-talos) [inline manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md]: Centos and other Linux (vs Unix, macOS) users get gmake by default as "make", so it's unnecessary and possibly confusing for newer users to call it gmake.

File: manual/Installationguide_ClamAV 0.99.2+_for_Ubuntu&Deb.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- Openssl version 0.9.8 or higher and libssl-devel packages
+- GCC compiler suite. (tested with 2.9x, 3.x and 4.x series)
+```
+If you are compiling with higher optimization levels than the default one (-O2
+for gcc), be aware that there have been reports of misoptimizations. The
+build system of ClamAV only checks for bugs affecting the default settings,
+it is your responsibility to check that your compiler version doesn’t have any
+bugs.
+
+- GNU make (gmake)
```
Comment (micah-at-talos) [inline manual/Installationguide_ClamAV 0.99.2+_for_Ubuntu&Deb.md]: Same comment as with CentOS wrt gmake is just called make on Linux.

File: manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+Once that finishes, run make install
+
+**Step 8:**
+I will start this step with a common error that might pop up when doing a freshclam
+![rhel-11](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/rhel/rhel-11.jpg)
+
+How do you fix that you ask? Simple:
+Run the following:
+Sudo ldconfig
```
Comment (micah-at-talos) [inline manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md]: lower-case "sudo"

File: manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
+Installation guide: ClamAV 0.99.2 +
```
Comment (micah-at-talos) [inline manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md]: It looks to me as though *all* of the screenshots for CentOS are actually from Ubuntu.  Not only does it say "Ubuntu" in several locations, but the distinct font and color scheme really give it away.

File: manual/Installation_Win_Guide-ClamAV_0.993+.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
+Installation guide: ClamAV 0.99. 3 +
+_*Note: some of the packages used for installation are for 99.3+ versions of ClamAV._
```
[the remark this answers is not in the document]
Comment (micah-at-talos) [reply manual/Installation_Win_Guide-ClamAV_0.993+.md]: Also worth noting that while the 0.99.3-beta's did use the "ClamAV-x64" directory name, 0.100.0-beta, 0.100.0-rc both use just "ClamAV" now.

File: manual/Installation_Mac_guide-ClamAV_0.99.2+.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+### Packages needed for installation:
+
+Below are the steps for installing ClamAV from source Mac OS X systems.
+```
+Required Packages:
+```
+- Zlib and zlib-devel packages
+- Openssl version 0.9.8 or higher and libssl-devel packages
+- GCC compiler suite. (tested with 2.9x, 3.x and 4.x series)
```
[the remark this answers is not in the document]
Comment (tmccourt) [reply manual/Installation_Mac_guide-ClamAV_0.99.2+.md]: This is straight from our clam documentation.

File: manual/Installation_Mac_guide-ClamAV_0.99.2+.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+![mac-1](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/Mac/mac-1.jpg)
+
+
+**Step 2:**
+You will want to look for the “Download” button in the upper left hand corner and click it.
+![mac-2](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/Mac/mac-2.jpg)
+
+**Step 3:**
+Once you are on the download section, you will see something similar to “The latest stable
+release is 0.99.2.” Underneath that, you will see tar.gz files. You will want to locate the one for
```
[the remark this answers is not in the document]
Comment (tmccourt) [reply manual/Installation_Mac_guide-ClamAV_0.99.2+.md]: "you will see something similar to "the latest stable release is 0.99.2" 

it's not saying that is the most stable, just something along those lines

File: manual/Installation_Mac_guide-ClamAV_0.99.2+.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+a flexible and scalable multi-threaded daemon, a command line scanner and advanced
+tool for automatic database updates. The core of the package is an anti-virus engine
+available in a form of shared library.
+
+The reason for this guide is to provide installation instructions to the ClamAV product.
+As of right now, you can install ClamAV on Linux, Mac OSX and Windows.
+
+### Packages needed for installation:
+
+Below are the steps for installing ClamAV from source Mac OS X systems.
```
[the remark this answers is not in the document]
Comment (vrtadmin) [reply manual/Installation_Mac_guide-ClamAV_0.99.2+.md]: This should be macOS.

File: manual/Installation_Mac_guide-ClamAV_0.99.2+.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (micah-at-talos) [reply manual/Installation_Mac_guide-ClamAV_0.99.2+.md]: Yup we should probs update our clam documentation as well.

File: manual/Installation_Mac_guide-ClamAV_0.99.2+.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+![mac-5](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/Mac/mac-5.jpg)
+
+**Step 6:**
+When you do a “ls” you will see the tar.gz file, and a folder for clamav-0.99.2.
+![mac-6](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/Mac/mac-6.jpg)
+
+You will want to “CD” into clamav-0.99.2:
+![mac-7](https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/pictures_4_markdown/Mac/mac-7.jpg)
+
+**Step 7** :
```
[the remark this answers is not in the document]
Comment (tmccourt) [reply manual/Installation_Mac_guide-ClamAV_0.99.2+.md]: This was from the guide, to add the --sysconfdir option
which is why I left it in for this guide (for now)

File: manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md (hunk 4 above)
Comment (tmccourt) [reply manual/Installation_guide-ClamAV 0.99.2+_rhel&centos.md]: screenshots are exactly the same for it. 
nothing I am doing for the commands differ from os to os.

**Pull request merged**